igvm/hypervisor.py

```diff
@@ -316,8 +316,10 @@ def format_vm_storage(self, vm, transaction=None):
     def validate_image_checksum(self, image):
         """Compares the local image checksum against the checksum returned by
         foreman."""
+
+        image_file = '{}/{}'.format(IMAGE_PATH, image)
         local_hash = self.run(
-            'md5sum {}/{}'.format(IMAGE_PATH, image)
+            "flock -x -w 30 {} -c 'md5sum {}'".format(image_file, image_file)
```

Comment:
You can use {1} in both places and only pass image_file once.

Reply:
{0} but I got the idea.